packages/app/src/client/services/PersonalContainer.js

```diff
@@ -2,7 +2,6 @@ import { Container } from 'unstated';
 
 import loggerFactory from '~/utils/logger';
 
-import { apiPost } from '../util/apiv1-client';
```

Comment:
Erase all diffs in this file

Reply:
This file is no longer included to this branch